packages/navigator/src/browser/navigator-frontend-module.ts

```diff
@@ -32,7 +32,13 @@ export default new ContainerModule(bind => {
     bind(CommandContribution).toDynamicValue(c => c.container.get(FileNavigatorContribution));
     bind(KeybindingContribution).toDynamicValue(c => c.container.get(FileNavigatorContribution));
     bind(MenuContribution).toDynamicValue(c => c.container.get(FileNavigatorContribution));
-    bind(MenuContribution).to(NavigatorMenuContribution).inSingletonScope();
+
+    bind(NavigatorMenuContribution).toSelf().inSingletonScope();
+    for (const serviceIdentifier of [KeybindingContribution, MenuContribution]) {
```

Comment:
Is it a pattern we use already?  Spelling out both lines like this:

    bind(KeybindingContribution).toService(NavigatorMenuContribution);
    bind(MenuContribution).toService(NavigatorMenuContribution);

would be more straightforward.

Reply:
I got "inspired" from [this](https://github.com/theia-ide/theia/blob/master/packages/terminal/src/browser/terminal-frontend-module.ts#L47) but maybe it's not worth it for two contributions only.

Reply:
Even then I think it would be better explicitly spelled out... it helps if you grep `bind\(MenuContribution\)` :)